R/convertNewClimate.R

```diff
@@ -353,6 +353,26 @@ convertNewClimate <- function(x, subtype, subset) { # nolint: object_name_linter
     x <- toolCountryFill(ghgFactor, fill = NA, verbosity = 2)
   }
 
+  if (grepl("RenShareTargets", subtype, fixed = TRUE)) {
+    # disaggregate NewClimate regions to iso-country level
+    # only dissaggregation for EU needed
+    # allocate EU shares also to EU countries
+    x_EU <- collapseDim(x["EU", , ])
+
+    # ISO-countries in EU region
+    EUR_NPi_countries <- c(
+      "POL", "CZE", "ROU", "BGR", "HUN", "SVK", "LTU", "EST", "SVN",
+      "LVA", "DEU", "FRA", "ITA", "ESP", "NLD", "BEL", "GRC", "AUT",
+      "PRT", "FIN", "SWE", "IRL", "DNK", "LUX", "CYP", "MLT", "JEY",
+      "FRO", "GIB", "GGY", "IMN", "HRV"
+    )
+
+    # add missing countries with NA
+    x <- toolCountryFill(x)
+    # assign share EU values to EU countries
+    x[EUR_NPi_countries, , ] <- x_EU
```

Comment:
As far as i can see, "EU" is not removed from `x`. Does this throw a warning, as "EU" is not part of the 249 ISO countries?

Reply:
Edit: please fix 
```
~~ WARNING: Data for following unknown country codes removed: EU
~~ NOTE:  - toolCountryFill set missing values for IMPORTANT countries to NA:
...
```

They just create span in the input data log